packages/framework/ini-files/config-specs.ini

```diff
@@ -2819,6 +2827,46 @@ use RHEL7_POST
 use rhel7_sems-gnu-8.3.0-openmpi-1.10.7-openmp_release-debug_static_no-kokkos-arch_no-asan_no-complex_no-fpic_mpi_no-pt_no-rdc_no-uvm_deprecated-on_no-package-enables
 use PACKAGE-ENABLES|ALL
 
+[rhel8_gcc-openmpi_debug_shared_no-kokkos-arch_no-asan_no-complex_no-fpic_mpi_no-pt_no-rdc_no-uvm_deprecated-on_no-package-enables]
+#uses a containerized environment
+use COMPILER|GNU
+use NODE-TYPE|SERIAL
+use BUILD-TYPE|DEBUG
+
+use RHEL8_LIB-TYPE|SHARED
+use KOKKOS-ARCH|NO-KOKKOS-ARCH
+
+use USE-ASAN|NO
+use USE-FPIC|NO
+use USE-MPI|YES
+use USE-PT|NO
+use USE-COMPLEX|NO
+use USE-RDC|NO
+use USE-UVM|NO
+use USE-DEPRECATED|YES
+use PACKAGE-ENABLES|NO-PACKAGE-ENABLES
+
+use COMMON_SPACK_TPLS
+
+opt-set-cmake-var MPI_EXEC_PRE_NUMPROCS_FLAGS                            STRING       : --bind-to;none
+opt-set-cmake-var Trilinos_ENABLE_COMPLEX_DOUBLE                         BOOL         : ON
```

Comment:
If we're enabling complex<double> then rename the entry from `no-complex` to `complex`.  I'm fine with either as long as it's consistent.

Reply:
@sebrowne thanks for catching that.  I have updated the build name to be consistent.  We should check the current gcc builds for consistency as well.  Most of this config is copied from an existing gcc 8.3.0 no_complex pr build